servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -144,13 +148,22 @@ public void processJournalEntry(JournalEntry entry) throws IOException {
       } catch (FileDoesNotExistException fdnee) {
         throw new RuntimeException(fdnee);
       }
+    } else if (entry instanceof InodePersistedEntry) {
+      InodePersistedEntry typedEntry = (InodePersistedEntry) entry;
```

Comment:
This entry is only used to mark directories are persisted? Perhaps it could be named differently? Currently it seems to overlap with `PersistFileEntry`.